test/integration/models/test_networking.py

```diff
@@ -16,36 +16,29 @@ def test_get_networking_rules(get_client, create_firewall):
 
 
 @pytest.mark.smoke
-def test_ip_addresses_share(self):
+def test_ip_addresses_share(get_client):
     """
     Test that you can share IP addresses with Linode.
     """
-    ip_share_url = "/networking/ips/share"
+    client = get_client
+    available_regions = client.regions()
+    chosen_region = available_regions[0]
+    label = "test-ip-share"
+
+    linode_instance, password = client.linode.instance_create(
+        "g5-standard-4",
+        chosen_region,
+        image="linode/debian9",
```

Comment:
Can we use the newer generation and smaller size of the Linode? Maybe also newer version of image?
like `g6-nanode-1` and "linode/debian12"